bicycleparameters/app-data/Benchmark/Parameters/BenchmarkBenchmark.txt

```diff
@@ -0,0 +1,26 @@
+w = 1.02+/-0.0
```

Comment:
The data files need to be in:

`bicycleparameters/app-data/bicycles/Benchmark/Parameters/BenchmarkBenchmark.txt`

You are missing `/bicycles/`.

Reply:
You're right. I updated the file structure. Let me know if there are any other problems with that. I'm still unable to run cycle_app.py from that directory and test for these issues. I ran it using as similar a method to you as I could, but still no dice.

```
xyzasdf@DESKTOP-GK4SQ6I MINGW64 ~/Desktop/bike-lab/BicycleParameters (cycle-app)
$ py bicycleparameters/cycle_app.py                                                                                     Traceback (most recent call last):
  File "bicycleparameters/cycle_app.py", line 7, in <module>
    import bicycleparameters as bp
ModuleNotFoundError: No module named 'bicycleparameters'
```